hydra-node/src/Hydra/Options.hs

```diff
@@ -550,6 +556,27 @@ hydraNodeCommand =
       (decodeUtf8 $ encodePretty Contract.scriptInfo)
       (long "script-info" <> help "Dump script info as JSON")
 
+defaultContestationPeriod :: ContestationPeriod
+defaultContestationPeriod = UnsafeContestationPeriod defaultContestationPeriod'
+
+defaultContestationPeriod' :: Natural
+defaultContestationPeriod' = 3
+
+contestationPeriodParser :: Parser Natural
+contestationPeriodParser =
+  option
+    auto
```

Comment:
Besides simple numeric values, this should also parse `show`n values of `ContestationPeriod`. Currently this is for example: `3s`.